python/example_code/lookoutvision/projects.py

```diff
@@ -0,0 +1,187 @@
+
+"""
+Amazon Lookout for Vision project code examples for using projects.
+Examples are used in the service documentation:
+https://docs.aws.amazon.com/lookout-for-vision/latest/developer-guide/.
+Shows how to create and delete a project. Also, how to get information about your projects.
+"""
+import logging
+from botocore.exceptions import ClientError
+
+
+logger = logging.getLogger(__name__)
+
+
+class Projects:
+    """
+    Provides example functions for creating, listing and deleting Amazon Lookout for Vision Projects
+    """
+
+    @staticmethod
+    def create_project(lookoutvision_client, project_name):
+        """
+        Creates a new Amazon Lookout for Vision project.
+        :param lookoutvision_client: An Amazon Lookout for Vision boto3 client.
+        :param  project_name: The name for the new project.
+        :return project_arn: The ARN of the new project.
+        """
+
+        try:
+            # Create a project
+            logger.info("Creating project: %s", project_name)
+            response = lookoutvision_client.create_project(ProjectName=project_name)
+
+            logger.info("project ARN: %s", response["ProjectMetadata"]["ProjectArn"])
+
+            return response["ProjectMetadata"]["ProjectArn"]
+
+        except ClientError as err:
+            logger.exception("Couldn't create project: %s", err.response["Message"])
+            raise
+
+    @staticmethod
+    def delete_project(lookoutvision_client, project_name):
+        """
+        Deletes an Amazon Lookout for Vision Model
+        :param lookoutvision_client: An Amazon Lookout for Vision boto3 client.
+        :param: project_name - the name of the project that you want to delete.
+        """
+        try:
+
+            # Delete a project
+            logger.info("Deleting project: %s", project_name)
+            response = lookoutvision_client.delete_project(ProjectName=project_name)
+            logger.info("Deleted project ARN: %s ", response["ProjectArn"])
+
+        except ClientError as err:
+            logger.exception("Couldn't delete project: %s", err.response["Message"])
+            raise
+
+    @staticmethod
+    def list_projects(lookoutvision_client):
+        """
+        Lists information about the projects in your account.
+        :param lookoutvision_client: An Amazon Lookout for Vision boto3 client.
+        """
+
+        try:
+            response = lookoutvision_client.list_projects()
+
+            for project in response["Projects"]:
+
+                print("Project: " + project["ProjectName"])
+                print("\tARN: " + project["ProjectArn"])
+                print("\tCreated: " + str(["CreationTimestamp"]))
+
+                print("Datasets")
+                project_description = lookoutvision_client.describe_project(
+                    ProjectName=project["ProjectName"]
+                )
+                if len(project_description["ProjectDescription"]["Datasets"]) == 0:
```

Comment:
Test for empty list just by testing for False:
```
if not project_description["ProjectDescription"]["Datasets"]:
    print('Nothin')
```

Reply:
Fixed.